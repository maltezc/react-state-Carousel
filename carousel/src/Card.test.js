import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

describe("Card tests", function () {
  // smoke test
  test("smoke test", function () {
    render(<Card caption="test caption" src="test caption" currNum={3} totalNum={3}/>)
  })

  // snapshot test
  test("matches snapshot", function () {
    const {container} = render(<Card caption="test caption" src="test caption" currNum={3} totalNum={3}/>)
    expect(container).toMatchSnapshot();
  })

})

