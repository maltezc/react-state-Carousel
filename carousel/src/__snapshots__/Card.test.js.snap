// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card tests matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test caption
    </h4>
    <img
      alt="test caption"
      class="Card-image"
      src="test caption"
    />
    <small
      class="Card-small"
    >
      Image 
      3
       of 
      3
      .
    </small>
  </div>
</div>
`;
